backend/prisma/schema.prisma

```diff
@@ -18,17 +18,35 @@ model User {
 }
 
 model ForkTemplate {
-  id    Int    @id @default(autoincrement())
-  url   String
-  owner String
-  repo String
-  forks Fork[]
+  id       Int    @id @default(autoincrement())
+  owner    String
+  repo     String
+  provider String
+
+  forks   Fork[]
+  actions ForkAction[]
+}
+
+model ForkAction {
+  key              String
+  githubActionName String
+  toState          String
```

Comment:
Defined which state the fork should be in after a successful action.

Reply:
Voisko toState:ssa, (ja ehkä näissä kaikissa?) käyttää primsan enumia stringin sijaan?

Reply:
Se olis aina paras! Laiskuus voitti...

Reply:
Pistetään tää muutos tulevaisuuden TODO-listalle...